package.json

```diff
@@ -278,7 +278,7 @@
     "@metamask/controller-utils": "^11.11.0",
     "@metamask/delegation-controller": "^0.6.0",
     "@metamask/delegation-deployments": "^0.11.0",
-    "@metamask/design-system-react": "^0.3.0",
+    "@metamask/design-system-react": "npm:@metamask-previews/design-system-react@0.3.0-preview.022dda1",
```

Comment:
publish preview of fixed react peer dependencies https://github.com/MetaMask/metamask-design-system/pull/809